File: app/src/main/java/com/ruslan/pocketdoc/data/Speciality.java
package com.ruslan.pocketdoc.data;

import com.google.gson.annotations.SerializedName;

public class Speciality {

    @SerializedName("Id")
    private String mId;

    @SerializedName("Name")
    private String mName;

    @SerializedName("BranchName")
    private String mBranchName;

    public String getId() {
        return mId;
    }

    public String getName() {
        return mName;
    }

    public String getBranchName() {
        return mBranchName;
    }
}

File: app/src/main/java/com/ruslan/pocketdoc/data/Speciality.kt
package com.ruslan.pocketdoc.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Speciality(
        @SerializedName("Id")
        @Expose
        val id: String,

        @SerializedName("Name")
        @Expose
        val name: String,

        @SerializedName("BranchName")
        @Expose
        val branchName: String
)